### PR TITLE
Updating the stock adjustement

### PR DESCRIPTION
**Pull request by DedrickEnc**
This PR add feature for writing in the posting journal after inserting data into voucher. so after making an adjustment the posting journal and voucher table will be updated.

### CODE REVIEW

File: client/src/modules/stock/adjustment/adjustment.html [hunk 1]
```diff
@@ -7,8 +7,8 @@
       </li>
       <li class="title" ng-if="StockCtrl.depot.uuid">
         <span>{{ StockCtrl.depot.text }}</span>
-        <span class="text-uppercase badge badge-primary" ng-show="StockCtrl.adjustmentOption">
-          {{ StockCtrl.adjustmentOption }}
+        <span class="text-uppercase badge badge-primary" ng-show="StockCtrl.adjustmentOption" translate>
+          {{ StockCtrl.adjustmentType}}
```
Comment (mbayopanda) [inline client/src/modules/stock/adjustment/adjustment.html:11]: 👍

**Review by mbayopanda — COMMENTED** [review]
This job works well, LGTM

**Comment by mbayopanda** [comment]
@DedrickEnc 

Could you update the receipt of adjustment by fixing the stock movement reference, it must be the same as in the journal

adjustment receipt fix

FYI this information is in the document_map table, which is map of stock movement document_uuid and document_map uuid.

**Comment by mbayopanda** [comment]
@DedrickEnc could you update this branch and fix those conflicts ?

**Comment by DedrickEnc** [comment]
@mbayopanda 
This P.R can be merged any time

File: client/src/modules/stock/adjustment/adjustment.html [hunk 2]
```diff
@@ -35,123 +35,121 @@
...
-          <bh-loading-button loading-state="StockCtrl.$loading" disabled="!StockCtrl.validForSubmit">
-            <span translate>FORM.BUTTONS.SUBMIT</span>
-          </bh-loading-button>
+      <!-- footer -->
+      <div class="row" style="margin-top: 5px;">
+        <div class="col-xs-6 col-xs-offset-6">
+          <div class="text-right">
+            <button class="btn btn-default" ng-click="StockCtrl.suspend(StockForm)" type="button" translate>
+              FORM.BUTTONS.SUSPEND
+            </button>
```
Comment (mbayopanda) [inline client/src/modules/stock/adjustment/adjustment.html:145]: [not blocking] : this button doesn't do anything

File: client/src/modules/stock/adjustment/adjustment.html [hunk 3]
```diff
@@ -35,123 +35,121 @@
...
+          <div class="btn-group pull-right" role="group">
+            <button
+              type="button"
+              class="btn btn-default"
+              ng-class="{'btn-primary' : StockCtrl.hasCacheAvailable }"
+              ng-click="StockCtrl.readCache()"
+              ng-disabled="!StockCtrl.hasCacheAvailable">
+              <span class="fa fa-recycle"></span>
+              <span translate>FORM.BUTTONS.RECOVER_ITEMS</span>
+            </button>
```
Comment (mbayopanda) [inline client/src/modules/stock/adjustment/adjustment.html:120]: [not blocking] : this button doesn't do anything as long as there is not any implementation of read and write into the cache for data defined in this page.

**Review by mbayopanda — APPROVED** [review]
LGTM

**Comment by jniles** [comment]
These were some nice commit messages :+1:

**Pull request merged**